nixos/modules/services/web-apps/keycloak.nix

```diff
@@ -272,6 +265,25 @@ in
         '';
       };
 
+      themeDevMode = {
```

Comment:
IMHO something like `disableThemeCache` is more suitable? Since it's only several config options used by a small number of developers, adding some instructions to `nixos/modules/services/web-apps/keycloak.md` might also be enough.